cms/models.py

```diff
@@ -10,10 +13,15 @@
 from cms.blocks import ResourceBlock
 
 
-class BootcampPage(Page):
+class BootcampProductPage(Page):
```

Comment:
Any specific reason for this rename? Bootcamps are the only products here so I guess we're fine with just "BootcampPage" unless you have another reason.

Reply:
not specific, Just for better clarity that we are pointing to product.

Reply:
I guess, name `Bootcamp` would be good enough as we have this the only product.